Src/PerformanceCollector/Shared/Implementation/QuickPulse/Helpers/QuickPulseQuotaTracker.cs

```diff
@@ -0,0 +1,135 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.QuickPulse
+{
+    using System;
+    using System.Threading;
+
+    internal class QuickPulseQuotaTracker
```

Comment:
Please make net new code doc commented and StyleCop compliant for future readability by others.